### DISCUSSION TITLE
Create 2024-25 season

### DISCUSSION THREAD

**Discussion by HyakuzaO** [kyleskom/NBA-Machine-Learning-Sports-Betting#448] [Q&A]
Hello guys ,

Can anyone please Share how to create the new season?

Thanks guys . Cheers

**Comment by t-dmac** [comment]
# Create dataset with the latest data for 2024-25 season
cd src/Process-Data
python -m Get_Data
python -m Get_Odds_Data
python -m Create_Games

# Train models
cd ../Train-Models
python -m XGBoost_Model_ML
python -m XGBoost_Model_UO

This is from the description

**Reply by HyakuzaO** [reply]
So, we dont need to do nothing in the files?

**Reply by wachna87** [reply]
we need some new files ... dataset 12_25 for example ... but the creator somehow dissappeared ...

**Reply by kyleskom** [reply]
You dont need to get the very latest data.

**Reply by vgella** [reply]
you can also edit the config.toml file to include the 2024-2025 season.

**Reply by Tanstones630** [reply]
> You dont need to get the very latest data.

Update the data until the end of the 24-25 season？ Does this data exclude playoff games?

**Comment by t-dmac** [comment]
even with roster changes ?

〽️ac

**Reply by kyleskom** [reply]
Player changes would be accounted for within those teams stats overall

**Comment by t-dmac** [comment]
Perfect.

Thanks for the response.

〽️ac

**Comment by t-dmac** [comment]
Any advice on steps for that?On Oct 29, 2024, at 5:34 AM, vgella ***@***.***> wrote:﻿
you just need to edit the config.toml file to include the 2024-2025 season.

**Reply by vgella** [reply]
my bad, it doesn't work. there's an already exists error when you try to define a 24 to 25 season that has start and end year 2024 in the odds.

**Reply by giannisni** [reply]
so how can we include new season? yes configuring  config.toml doesnt change it

**Comment by mendez97cr** [comment]
Message me I’d like to work with you. I can get the latest NBA data and
probably add it to the ML pipeline.

**Reply by vgella** [reply]
Ok email me at venkat@thirdeyeai.org
